### PR TITLE
Add tool to automaticaly update status of AIP-47 issues.

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragement file, named `{pr_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by potiuk** [comment]
cc: @bhirsz

**Comment by potiuk** [comment]
A little house-keeping tool to keep AIP-47 issues up-to-date semi- automatically :)

File: dev/system_tests/README.md
Comment (bhirsz) [inline dev/system_tests/README.md]: ``deve`` typo

File: dev/system_tests/README.md
Comment (bhirsz) [inline dev/system_tests/README.md]: `:` suggest there will be code snippet?

File: dev/system_tests/update_issue_status.py
Comment (bhirsz) [inline dev/system_tests/update_issue_status.py]:
```suggestion
        You can omit it if you have a GITHUB_TOKEN env variable set.
```

File: dev/system_tests/update_issue_status.py
Comment (bhirsz) [inline dev/system_tests/update_issue_status.py]: Do we use logger here?

**Comment by potiuk** [comment]
All corrected :) I also found that the github link scope was read-only (copied from read-only piece of code). Changed it to write scope.

**Comment by potiuk** [comment]
Nice tool waits for an approval :)

**Review by bhirsz — APPROVED** [review]

**Comment by bhirsz** [comment]
Nice! Good to see tools such as this - it's those small scripts that sometimes matters the most ;)

**Comment by potiuk** [comment]
Would be great to merge it so that I could run it every time I merge an AIP-47 change without rebasing it :)

**Comment by potiuk** [comment]
(but it works nicely BTW)

**Comment by potiuk** [comment]
BTW. @bhirsz @ferruzzi  added some stats.

Excerpt:

```
 1176814835: Migrate Google system tests to new design

Summary of performed actions: for Migrate Google system tests to new design
   Re-added file number (still there): 0
   Completed file number: 0
   Done 14/71 = 19.72%

 1176885644: Migrate Amazon example DAGs to new design

Summary of performed actions: for Migrate Amazon example DAGs to new design
   Re-added file number (still there): 0
   Completed file number: 0
   Done 8/37 = 21.62%

Summary of ALL performed actions: for 44 issues
   Re-added file number: 0
   Completed file number: 0
   Done 39/275 = 14.18%

```

**Comment by potiuk** [comment]
BTW. @bhirsz @ferruzzi - you will be able to run the tool in `--dry-mode` once merged and see the stats and what will be updated  when committer runs it with read-only token or even without it.

**Comment by ferruzzi** [comment]
Very cool :sunglasses:

**Comment by potiuk** [comment]
@mnojek - now we got to the full potential of the issue management

**Review by eladkal — APPROVED** [review]

**Pull request merged**
